src/isIsogram.test.js

```diff
@@ -6,4 +6,31 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it('should handle empty string', () => {
+    expect(isIsogram('')).toBe(true);
+  });
+
+  it('should handle single character string', () => {
+    expect(isIsogram('a')).toBe(true);
+  });
+
+  it('should handle repeated characters', () => {
+    expect(isIsogram('look')).toBe(false);
+    expect(isIsogram('Adam')).toBe(false);
+  });
+
+  it('should handle non-alphabetic characters', () => {
+    expect(isIsogram('playgrounds!')).toBe(true);
+    expect(isIsogram('123')).toBe(true);
+  });
```

Comment:
The test case for non-alphabetic characters assumes that such characters should be ignored, and the string should still be considered an isogram if the alphabetic characters do not repeat. Ensure that the `isIsogram` function implementation aligns with this assumption.